viewing_party/main.py

```diff
@@ -0,0 +1,139 @@
+def create_movie(title,genre,rating):
+    new_movie = {
+        "title": "",
+        "genre": "",
+        "rating": 0
+    }
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return None
+
+
+def add_to_watched(user_data,movie):
+    for key, value in user_data.items():
+        user_data[key].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    for key, value in user_data.items():
+        user_data[key].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movies in user_data["watchlist"]:
+        if title == movies["title"]:
+            user_data["watched"].append(movies)
+            user_data["watchlist"].pop(0)
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    rating_list = []
+    if user_data["watched"]:
+        for movies in user_data["watched"]:
+            rating_list.append(movies["rating"])
+        return sum(rating_list)/len(rating_list)
+    else:
+        return 0.0
+
+def get_most_watched_genre(user_data):
+    word_counter = {}
+    genre_list = []
+    if user_data["watched"]:
+        for movies in user_data["watched"]:
+            genre_list.append(movies["genre"])
+        for genres in genre_list:
+            if genres in word_counter:
+                word_counter[genres] += 1
+            else:
+                word_counter[genres] = 1
+        most_watched = sorted(word_counter, key = word_counter.get, reverse = True)
+        return most_watched[0]
+    else:
+        return None
+
+#get_friends_movie_list() is a helper function that iterates through the user_data's friend movie list
+def get_friends_movie_list(user_data):
+    friends_movie_list = []
+    for friends_watched_movies in user_data["friends"]:
+        for watched, movies in friends_watched_movies.items():
+            friends_movie_list.append(movies)
+    comprehensive_friend_list = (friends_movie_list[0] + friends_movie_list[1])
+    return comprehensive_friend_list
```

Comment:
This is a great helper function, however, I am curious as to why you're adding specific movies by id to the comprehensive_friend_list.